openquake/calculators/extract.py

```diff
@@ -51,6 +51,7 @@
 I64 = numpy.int64
 F32 = numpy.float32
 F64 = numpy.float64
+TWO30 = 2 ** 32
```

Comment:
~`2 ** 32`~ `2 ** 30`

Reply:
possibly the source of the mismatch between the rupture ids used internally vs the rupture ids displayed in the exported files